test/integration/integration_test.go

```diff
@@ -555,3 +583,18 @@ func proxyLogShouldContain(expected string) error {
 func proxyLogShouldContainContent(expected *gherkin.DocString) error {
 	return compareExpectedWithActualContains(expected.Content, testProxy.GetLog())
 }
+
+func CatDockerConfigFile() error {
```

Comment:
👍 If this function is not needed to be access outside of this scope then better to name it as `catDockerConfigFile()`